libbeat/processors/add_process_metadata/config.go

```diff
@@ -72,9 +88,14 @@ func init() {
 
 func defaultConfig() config {
 	return config{
-		IgnoreMissing:    true,
-		OverwriteKeys:    false,
-		RestrictedFields: false,
+		IgnoreMissing:         true,
+		OverwriteKeys:         false,
+		RestrictedFields:      false,
+		MatchPIDs:             []string{"process.pid", "process.ppid", "process.parent.pid", "process.parent.ppid"},
+		HostPath:              "/",
+		CgroupPrefixes:        []string{"/kubepods", "/docker"},
+		CgroupCacheExpireTime: 300,
```

Comment:
How about using `cacheExpiration` as the default here? This is the cache we use for the other parts of proc